Formula/kubevela.rb

```diff
@@ -18,8 +18,13 @@ class Kubevela < Formula
   depends_on "go" => :build
 
   def install
-    system "make", "vela-cli", "VELA_VERSION=#{version}"
-    bin.install "bin/vela"
+    ldflags = %W[
+      -s -w
+      -X github.com/oam-dev/kubevela/version.VelaVersion=#{version}
+      -X github.com/oam-dev/kubevela/version.GitRevision=#{Utils.git_head}
+    ].join(" ")
+
+    system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"vela", "./references/cmd/cli"
   end
 
   test do
```

Comment:
Can we add a test to check the version given that we're passing it via `ldflags` now?

Reply:
done